.travis.yml

```diff
@@ -9,10 +11,16 @@ script: ./travis-run-tests.sh
 before_deploy:
   - python setup.py compile_catalog
   - openssl aes-256-cbc -K $encrypted_f6aff9134f80_key -iv $encrypted_f6aff9134f80_iv -in openfisca_bot.enc -out openfisca_bot -d
+  - openssl aes-256-cbc -K $encrypted_a272d9ca0751_key -iv $encrypted_a272d9ca0751_iv -in deploy-api.enc -out deploy-api -d
+  - eval "$(ssh-agent -s)"
+  - chmod 400 ./openfisca_bot ./deploy-api
+  - ssh-add ./openfisca_bot ./deploy-api
 deploy:
   - provider: script
     skip_cleanup: true
-    script: ./travis-release-tag.sh
+    script:
+      - git tag "$(python setup.py --version)"
```

Comment:
I'd be more confortable if this was tested on a test branch, with a dev version.